packages/neuron-ui/src/widgets/DatetimePicker/index.tsx

```diff
@@ -49,7 +40,9 @@ const DatetimePicker = ({
   const tomorrow = new Date()
   tomorrow.setDate(tomorrow.getDate() + 1)
   const isSinceTomorrow = display ? new Date(display).getTime() >= new Date(tomorrow.toDateString()).getTime() : false
-  const disabled = !selected || !isSinceTomorrow
+  const isDayEqualsForPreset =
+    preset != null && selected != null && new Date(+preset).toDateString() === selected.toDateString()
+  const disabled = !selected || !isSinceTomorrow || isDayEqualsForPreset
```

Comment:
Why does `isDayEqualsForPreset` disable `save`

Reply:
Oh, this is to implement `We should have the 'Save' button initially disabled until the user manually makes a change.`. I'm not sure if we still need this after displaying the locked date on the save button. @Danie0918 , what do you think?

Reply:
It depends on whether our default date is a time commonly used by users, and if so, we can provide options. Conversely, if the user does not have a universal lock time, it is best not to allow saving without manual selection to avoid misuse.

Reply:
After discussing with @Danie0918 , it was decided to remove this condition to avoid confusion when the user selects a different date and then reverts it, resulting in the button being disabled.